src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -13792,7 +13823,11 @@ internal string TraceString()
                            _statisticsIsInTransaction ? bool.TrueString : bool.FalseString,
                            _fPreserveTransaction ? bool.TrueString : bool.FalseString,
                            _connHandler == null ? "(null)" : _connHandler.ConnectionOptions.MultiSubnetFailover.ToString((IFormatProvider)null),
+#if NETFRAMEWORK
```

Comment:
Probably want to remove the line fore this?

Reply:
Afraid not - this is designed to handle the TransparentNetworkIPResolution flag which only exists as a legacy setting in netfx. There are a few places where we have to conditionally compile to account for this.